### PR TITLE
Refactor part 1

### PR DESCRIPTION
**Pull request by JarJak**

### CODE REVIEW

File: public/theme/skeleton/partials/_sub_fields.twig [hunk 1]
```diff
@@ -24,7 +24,7 @@
     {%- spaceless %}
         {# Set up the array of fieldnames that should be iterated. We do this by looping
            over _all_ the fields, and skipping those in the 'omittedkeys' array. #}
-        {% set omittedkeys = [ 'id', 'slug', 'datecreated', 'datechanged', 'datepublish',
+        {% set omittedkeys = [ 'id', 'slug', 'image', 'datecreated', 'datechanged', 'datepublish',
```
[the remark this answers is not in the document]
Comment (JarJak) [reply public/theme/skeleton/partials/_sub_fields.twig]: it was displaying raw image url

[the remark this answers is not in the document]
Comment (bobdenotter) [reply public/theme/skeleton/partials/_sub_fields.twig]: @JarJak ah, right!! Feel free to leave it like this, and I'll fix it as a followup to #232 :-)

**Comment by JarJak** [comment]
Needs to merge-refactor changes in #232

File: public/theme/skeleton/index.twig [hunk 2]
```diff
@@ -19,4 +19,13 @@
 
     {% endif %}
 
+    <hr>
```
Comment (bobdenotter) [inline public/theme/skeleton/index.twig]: This is a revert of https://github.com/bolt/four/commit/bf0f9eab211181712d405e9fa279e2f942c1b21a#diff-f6a467f42f0eec03b12d623560b5d5aaL26 , and is not needed anymore

Comment (JarJak) [reply public/theme/skeleton/index.twig]: removed

**Pull request merged**